agent/bench-scripts/tests/pbench-user-benchmark/test-67.pre

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Remove the mocked start/stop tool scripts
+rm ${_testopt}/unittest-scripts/pbench-*-tools 
+
+# Create an empty tools directory.
+mkdir ${_testdir}/tools-v1-empty
```

Comment:
Did you want, in this test or in another one, to try using the approach of registering-then-clearing the default tool set, rather than this hack of manually creating an empty directory?

Reply:
I don't think that is necessary because the `ToolGroup` code works independent of a name.

This was the path of least resistance where I did not have to undo the provided `default` tool group directory in any way.

This is a very simple step for users to take for now.  I'd rather not have to have an empty directory at all.  But that will mean a bit more changes to how we handle tool groups.